tests/test_integration.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from taar.context import default_context
+from taar import ProfileController, ProfileFetcher
+from taar import recommenders
+import time
+
+
+def create_recommendation_manager():
+    root_ctx = default_context()
+    client = ProfileController('us-west-2', 'taar_addon_data_20180206')
+    pf = ProfileFetcher(client)
+    root_ctx['profile_fetcher'] = pf
+    r_factory = recommenders.RecommenderFactory(root_ctx.child())
+    root_ctx['recommender_factory'] = r_factory
+    rm = recommenders.RecommendationManager(root_ctx.child())
+    return rm
+
+
+@pytest.mark.skip("This is an integration test")
```

Comment:
Can you explain your thinking here around writing a new test but skipping it.

If we don't want to run it on CI, we can skip it there. 

If we don't want to run it by default we can add a pytest mark and update setup.cfg to exclude that mark by default.

I'm concerned that this test will never be run and will bitrot.

Reply:
The function here is mostly a smoke test.  We've had problems where we missed something that unit tests could not adequately capture.  To run an end-to-end test, a person needs to have proper S3 credentials to read the JSON blobs from the dev environment.  

bitrot is a real concern - we can have ops provision a special CI access key for us to read the JSON blobs for each of the recommenders when we run under test.  

@mlopatka - i don't imagine there's a privacy concern here, but is that ok?    

Basically - anyone who uses our CI access key would have *read* permission on each of the JSON blobs for the recommenders.

Reply:
okay. lets open an issue. there should be a way to make this work. i've worked with similar problems before - e.g. you only want certain committers to be able to upload to your s3 bucket.